### ISSUE TITLE
When fix patch is applied to my dataset, the dataset is unbalanced. Why does the training set perform well, but the loss of verification set first decreases and then keeps rising?

### ISSUE THREAD

**Issue by a571-op** [kekmodel/FixMatch-pytorch#24]
When fix patch is applied to my dataset, the dataset is unbalanced. Why does the training set perform well, but the loss of verification set first decreases and then keeps rising?

**Comment by arfeen93** [comment]
Same thing is happening to me.

**Comment by 15510112867** [comment]
> When fix patch is applied to my dataset, the dataset is unbalanced. Why does the training set perform well, but the loss of verification set first decreases and then keeps rising?

Friend，I meet the problem too，have you ever solved？

**Comment by jingyang2017** [comment]
same here.
